src/test/java/com/wikia/webdriver/common/dataprovider/ads/FandomAdsDataProvider.java

```diff
@@ -53,40 +60,41 @@ private FandomAdsDataProvider() { }
     };
   }
 
-    @DataProvider
-    public static Object[][] vuapPage() {
-        return new Object[][]{
-                {
-                        AdsFandomTestTemplate.PAGE_TYPE_ARTICLE,
-                        FandomAdsDataProvider.VUAP_PAGE_SLUG,
-                        AdsFandomContent.GPT_TOP_LEADERBOARD,
-                        String.format(AD_UNIT_TEMPLATE, AdsFandomContent.TOP_LEADERBOARD)
-                },
-                {
-                        AdsFandomTestTemplate.PAGE_TYPE_ARTICLE,
-                        VUAP_PAGE_SLUG,
-                        AdsFandomContent.GPT_BOTTOM_LEADERBOARD,
-                        String.format(AD_UNIT_TEMPLATE, AdsFandomContent.BOTTOM_LEADERBOARD)
-                }
-        };
-    }
+  @DataProvider
+  public static Object[][] vuapPage() {
+    return new Object[][]{
+        {
+            AdsFandomTestTemplate.PAGE_TYPE_ARTICLE,
+            FandomAdsDataProvider.VUAP_PAGE_SLUG,
+            AdsFandomContent.TOP_LEADERBOARD,
+            String.format(AD_UNIT_TEMPLATE, AdsFandomContent.TOP_LEADERBOARD)
+        },
+        {
+            AdsFandomTestTemplate.PAGE_TYPE_ARTICLE,
+            VUAP_PAGE_SLUG,
+            AdsFandomContent.BOTTOM_LEADERBOARD_DESKTOP,
+            String.format(AD_UNIT_TEMPLATE, AdsFandomContent.BOTTOM_LEADERBOARD)
+        }
+    };
+  }
 
-    @DataProvider
-    public static Object[][] vuapPageMobile() {
-        return new Object[][]{
-                {
-                        AdsFandomTestTemplate.PAGE_TYPE_ARTICLE,
-                        FandomAdsDataProvider.VUAP_PAGE_SLUG,
-                        AdsFandomContent.GPT_TOP_LEADERBOARD,
-                        String.format(AD_UNIT_TEMPLATE, AdsFandomContent.TOP_LEADERBOARD)
-                }
-//  Currently on Fandom mobile BFFA is not displayed ADEN-4479 waiting for fix
```

Comment:
[ADEN-4479](https://wikia-inc.atlassian.net/browse/ADEN-4479) is closed. Are we fine with uncommenting this part?